src/media/css/detail/base.styl

```diff
@@ -195,3 +193,11 @@
         display: block;
     }
 }
+
+.app-report-abuse .button {
+    display: block;
+    margin: 0 auto;
+    max-width: 100%;
+    text-align: center;
+    width: 300px;
+}
```

Comment:
It will usually be 300px. Unless the containing element shrinks to a width below 300px, in which case the button will take up 100%.